paz/datasets/fat.py

```diff
@@ -25,16 +24,23 @@ class FAT(Loader):
             Estimation (DOPE)](https://github.com/NVlabs/Deep_Object_Pose)
     """
     # TODO: Allow selection of class_names.
-    def __init__(self, path, split='train', class_names='all'):
+    def __init__(self, path, split=(0.4, 0.4, 0.2), class_names = 'all'):
```

Comment:
the variable split is only intended to set to be 'train', 'validation' or 'test'. Please don't change that behavior.